docs/src/explanations/connectable.md

```diff
@@ -163,21 +163,21 @@ Because `alignedChild` and `flippedChild` are non-aggregates, `Input` is basical
 ```scala mdoc:silent
 import chisel3._
 class ParentWithOutputInput extends Bundle {
-  val alignedCoerced = Output(UInt(32.W)) // Equivalent to just UInt(32.W)
```

Comment:
i think this is indeed trying to talk about coerced, as its discussing Input/Output not Flipped/Aligned though its not very interesting because it's just a UInt inside... It's trying to get your mental model right for the next example where the coerced does matter. So can you roll back this section's change of coerced to child?